Assets/00_Content/Scripts/Interactables/BeerTap.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+namespace Interactables {
+
+	public class BeerTap : Interactable {
+
+		[SerializeField] private GameObject beerPrefab;
+		[SerializeField] private Transform beerSpawnpoint;
+
+		public void Start() {
+
+			onInteraction += HandleOnInteraction;
```

Comment:
Subscribing to an event on the base class doesn't feel right to me. Would overriding a virtual method be an option?

Reply:
Agreed, it makes way more sense lol